crates/stages/src/stages/tx_lookup.rs

```diff
@@ -175,7 +177,9 @@ impl<DB: Database> Stage<DB> for TransactionLookupStage {
             }
         }
 
-        Ok(UnwindOutput { stage_progress: input.unwind_to })
+        let unwind_progress = *range.start() - 1;
```

Comment:
might be worth making it one of the returned values of `unwind_block_range_with_threshold`

Reply:
good point, fixed